internal/backend/svc/svc.go

```diff
@@ -172,7 +172,13 @@ func makeFxOpts(cfg *Config, opts RunOptions) []fx.Option {
 		Component("projects", configset.Empty, fx.Provide(projects.New)),
 		Component("projectsgrpcsvc", projectsgrpcsvc.Configs, fx.Provide(projectsgrpcsvc.New)),
 		Component("envs", configset.Empty, fx.Provide(envs.New)),
-		Component("vars", configset.Empty, fx.Provide(vars.New)),
+		Component(
+			"vars",
+			configset.Empty,
+			fx.Provide(vars.New),
+			fx.Provide(func(v *vars.Vars) sdkservices.Vars { return v }),
```

Comment:
is this change exposing Vars type instead of vars service just to inject connections service ?

Reply:
in order to expose `SetConnections` as well, used below.